user-interface/src/main/java/life/qbic/datamanager/views/projects/project/samples/SampleContentComponent.java

```diff
@@ -85,6 +102,56 @@ private void reloadOnBatchRegistration() {
         event -> displayComponentInContent(sampleDetailsComponent));
   }
 
+  private void addBatchUpdateListeners() {
```

Comment:
please inline this method. As you extracted methods for the listeners, I think this method should be inlined. It has no clear responsibility and the calls to addXYZListener are more telling.
Alternatively split this method into separate method for the separate types of events. This would lead to a better separation of concerns. You would end up with a bunch of one-liner increasing the complexity of the code without adding information that cannot be retrieved from the one line they hide.

Reply:
I moved the Listeners to the constructor